extensions/big-data-cluster/src/bigDataCluster/controller/clusterController.ts

```diff
@@ -0,0 +1,126 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { EndpointRouterApi } from './apiGenerated';
+// import { Socket } from 'net';
+// import { IncomingHttpHeaders } from 'http';
+
+export class ClusterController {
+
+	public async getEndPoints(
```

Comment:
this can be a simple `export function getEndpoints` - no need for a class as it's not stateful. Also note the casing of `getEndpoints` not `getEndPoints` as endpoint is a single word.

Reply:
As mentioned, alternative is have username/password in constructor (so in future, would replace with alternative auth and the same `getEndpoints` API would work regardless of Basic vs Kerberos auth.

Reply:
changed to "export function getEndpoints"